dist/index.d.ts

```diff
@@ -135,16 +135,27 @@ export type StatsMessage = MessageBase & {
 	stats: Stats;
 };
 export type TransactionSegmentStats = {
-	count: number;
+	txnCount: number;
 	value: number;
-	gasUsed: number;
 };
 export type HomepagePendingMessage = MessageBase & {
```

Comment:
Since this library (and API) may get consumed by other projects, maybe we should name this differently. Something like `MempoolSummary` and `BlockSummary`?

Reply:
Good call!